omod/src/main/webapp/resources/scripts/relationships/relationships.js

```diff
@@ -8,11 +8,24 @@ angular.module('relationships', ['relationshipTypeService', 'relationshipService
         $scope.relationshipTypes = [];
 
         $scope.relationships = [];
+        
+        $scope.personDetails = new Array();
 
         $scope.init = function(personUuid, excludeRelationshipTypes) {
             $scope.thisPersonUuid = personUuid;
             RelationshipService.getRelationships({ v: 'default', person: personUuid }).then(function(result) {
                 $scope.relationships = result;
+                
+                //get datails about the related person
```

Comment:
It should not be necessary to do this. You ought to be able to just change the call a few lines above to request

    v: 'full'

This will return a bigger representation of each relationship, including the more details of the related person